ci/tests/general/general_alba.py

```diff
@@ -630,36 +630,56 @@ def filter_disks(disk_names, amount, disk_type):
         :param disk_type: Type of disk
         :return: Filtered disks
         """
-        count = 0
         node_ids = []
-        filtered_disks = dict()
+        list_of_available_disks = {}
+        filtered_disks = {}
+        disk_count = 0
+        # disk_names = dictionary with node_ids as keys and values as a list of uninitialised disk names
         for node_id in disk_names.iterkeys():
```

Comment:
put an example dict here - that makes things more clear in comparison with an explanation